iocore/eventsystem/UnixEvent.cc

```diff
@@ -104,3 +106,19 @@ Event::schedule_every(ink_hrtime aperiod, int acallback_event)
     ethread->EventQueueExternal.enqueue_local(this);
   }
 }
+
+#ifdef ENABLE_EVENT_TRACKER
+
+void
+Event::set_location()
+{
+  _location = ink_backtrace(3);
+}
+
+const void *
+Event::get_location() const
```

Comment:
I don't see anything calling `get_location`. Does anything use the location yet? Or is the plan to do that in a future PR?

Reply:
Right. It's just the buddy of `set_location` for now. 
A possible use case is printing the location on debug log instead of using `add2line` command.